frontends/web/src/api/backend.ts

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
The filename `api/backend.ts` is strange. Are all API calls moving here at some point? Maybe @thisconnect knows more.

Reply:
I just put it here because it is direct backend/handlers.go endpoint with no namespace.

Reply:
I see. So, the plan is to move here all the other API endpoints which are in backend/handlers.go?

Reply:
Maybe just `coins.ts` and `apiGet('coins')`?

Reply:
one file per function is excessive?

Reply:
just wanted to give some options, I didn't mean to suggest to write a file per function.